libs/shared/infrastructure/di/src/lib/local-storage-infrastructure-container.ts

```
@@ -21,15 +21,20 @@
+  static getUsersRepository() {
     if (!this.usersRepository) {
-      this.usersRepository = new UsersRepositoryImpl(
-        this.getLocalStorageRepository(),
-      );
+      this.usersRepository = new UsersRepositoryImpl(this.getLocalStorageRepository()));
     }
     return this.usersRepository;
   }
 
+  static getTasksRepository(): TasksRepository {
+    if (!this.tasksRepository) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)